bin/wait-elasticsearch

```diff
@@ -26,12 +26,13 @@
 const yargs = require('yargs');
 const { Client: ESClient } = require('@elastic/elasticsearch');
 
+const config = require('../lib/config');
 const Kuzzle = require('../lib/kuzzle');
 
 const sleep = seconds => new Promise(resolve => setTimeout(() => resolve(), seconds * 1000))
 
 async function waitForEs (maxSeconds = 120) {
-  const kuzzle = new Kuzzle();
+  const kuzzle = new Kuzzle(config);
```

Comment:
The Kuzzle constructor doesn't take arguments.

Reply:
Ah yes not on master :expressionless:

Reply:
And on 2-dev the wait-elasticsearch script will soon be removed too (see #1754) :grin: